apps/console/app/routes/_shell/sidebar-playground.tsx

```
@@ -0,0 +1,25 @@
+import { Chat } from "@hebo/aikit-ui/blocks/Chat";
+
+type Branch = {
+  models?: Array<{
+    alias: string;
+    type: string;
+  }>;
+};
+
+export function PlaygroundSidebar({ activeBranch }: { activeBranch?: Branch }) {
+  const gatewayBaseUrl = String(import.meta.env.VITE_GATEWAY_URL ?? "");
```

You are hiding an error condition (aka VITE_GATEWAY_URL is not set). Log an error.